packages/dataviews/src/field-types/text.tsx

```diff
@@ -63,6 +63,8 @@ function Edit< Item >( {
 
 		return (
 			<SelectControl
+				__next40pxDefaultSize
+				__nextHasNoMarginBottom
```

Comment:
This can be seen in the "DataViews / DataForm" story in Storybook.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/319fda4f-bcc8-40e4-a881-03521e1f5706" alt="DataForm, before" width="720">|<img src="https://github.com/user-attachments/assets/3d59eb09-d94b-4932-91e7-aa7f938e0147" alt="DataForm, after" width="720">|

Reply:
Addressed in #64324